### PR TITLE
use patched react-native-screens to fix keyboard getting stuck open

### PR DESCRIPTION
**Pull request by buoyad**
See https://github.com/kmagiera/react-native-screens/pull/77 and https://github.com/keybase/react-native-screens/commit/0465a795e7bb59637a76857f11e71c2882341eb3. r? @keybase/react-hackers

### CODE REVIEW

File: shared/yarn.lock [hunk 1]
```diff
@@ -12772,11 +12772,17 @@ react-native-safe-module@^1.1.0:
   dependencies:
     dedent "^0.6.0"
 
-react-native-screens@1.0.0-alpha.22, "react-native-screens@^1.0.0 || ^1.0.0-alpha":
+"react-native-screens@^1.0.0 || ^1.0.0-alpha":
   version "1.0.0-alpha.22"
```
Comment (chrisnojima) [inline shared/yarn.lock]: why does this still exist

Comment (buoyad) [reply shared/yarn.lock]: Dunno, I `yarn install`ed and this stayed. is there a better command to update the lockfile?

**Comment by buoyad** [comment]
@chrisnojima `react-native-screens` is still there because `react-navigation` depends on it. `debounce` was added because `react-native-screens` depends on it.

File: shared/yarn.lock (hunk 1 above)
Comment (chrisnojima) [reply shared/yarn.lock]: lets use resolutions to force it over

**Pull request merged**